dist/less/ds6/variables.less

```diff
@@ -3,43 +3,98 @@
 // Colours
 //----------------------
 
-@ds6-color-g201-gray: #fff;
```

Comment:
We should leave these variables in place for backwards compatibility.

Reply:
When adding them back in, please also add a comment that they are deprecated.

Reply:
I actually thought of that just as I was submitting the PR. Already in the works.